src/web/server/htmlTemplate.ts

```diff
@@ -110,32 +115,32 @@ export const htmlTemplate = ({
-           GGGGGGGG             GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG          
-          GGGGGGGG              GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG         
-          GGGGGGGG              GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG         
-          GGGGGGGG              GGGGGGGGGGGG              GGGGGGGGGGGGG         
-           GGGGGGG              GGGGGGGGGGGGG           GGGGGGGGGGGGGG          
-           GGGGGGGG             GGGGGGGGGGGGG           GGGGGGGGGGGGGG          
-            GGGGGGGG            GGGGGGGGGGGGG           GGGGGGGGGGGGG           
-             GGGGGGGG            GGGGGGGGGGGG           GGGGGGGGGGGG            
-              GGGGGGGGG           GGGGGGGGGGG           GGGGGGGGGGG             
-                GGGGGGGGGG         GGGGGGGGGG           GGGGGGGGG               
-                  GGGGGGGGGGG        GGGGGGGG        GGGGGGGGGG                 
-                    GGGGGGGGGGGGGG      GGGGG  GGGGGGGGGGGGGG                   
-                       GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG                      
-                            GGGGGGGGGGGGGGGGGGGGGGGGG                           
-                                    GGGGGGGGG                                   
-                                              
+                                    GGGGGGGGG
+                           GGGGGGGGGGGGGGGGGGGGGGGGGG
+                       GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG
+                    GGGGGGGGGGGGGGGGG      GG   GGGGGGGGGGGGG
+                  GGGGGGGGGGGG        GGGGGGGGG      GGGGGGGGGG
+                GGGGGGGGGGG         GGGGGGGGGGGGG       GGGGGGGGG
+              GGGGGGGGGG          GGGGGGGGGGGGGGGGG     GGGGGGGGGGG
+             GGGGGGGGG           GGGGGGGGGGGGGGGGGGG    GGGGGGGGGGGG
+            GGGGGGGGG           GGGGGGGGGGGGGGGGGGGGGG  GGGGGGGGGGGGG
+           GGGGGGGGG            GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG
+           GGGGGGGG             GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG
+          GGGGGGGG              GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG
+          GGGGGGGG              GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG
+          GGGGGGGG              GGGGGGGGGGGG              GGGGGGGGGGGGG
+           GGGGGGG              GGGGGGGGGGGGG           GGGGGGGGGGGGGG
+           GGGGGGGG             GGGGGGGGGGGGG           GGGGGGGGGGGGGG
+            GGGGGGGG            GGGGGGGGGGGGG           GGGGGGGGGGGGG
+             GGGGGGGG            GGGGGGGGGGGG           GGGGGGGGGGGG
+              GGGGGGGGG           GGGGGGGGGGG           GGGGGGGGGGG
+                GGGGGGGGGG         GGGGGGGGGG           GGGGGGGGG
+                  GGGGGGGGGGG        GGGGGGGG        GGGGGGGGGG
+                    GGGGGGGGGGGGGG      GGGGG  GGGGGGGGGGGGGG
+                       GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG
+                            GGGGGGGGGGGGGGGGGGGGGGGGG
+                                    GGGGGGGGG
+
```

Comment:
Won’t this break the rendering?

In any case it’s wonderfully lean.

Reply:
Looks okay to me 👍 

![Screenshot 2021-03-23 at 17 33 42](https://user-images.githubusercontent.com/1336821/112191461-f34f9d80-8bfd-11eb-8dbe-d25c6d3535d1.jpg)

Reply:
That said, for the sake of a tidy git history we should probably remove these changes. Not sure how they got here, is prettier getting overzelous again?

Reply:
I think @ioanna0 uses IntelliJ, could this be it?

Reply:
Arghh, I also though it was because I'm using IntelliJ but VS also changes it on Save. Could it be some prettier settings have changed since the last time this file was edited?